docs/data/material/customization/theming/theming.md

```diff
@@ -10,7 +10,7 @@ To promote greater consistency between apps, light and dark theme types are avai
 
 ## Theme provider
 
-Use `ThemeProvider` or [`CssVarsProvider`](#css-variables-provider) to inject a custom theme into your application.
+Use `ThemeProvider` to inject a custom theme into your application.
```

Comment:
Not related to this PR, but the "However" sounds weird to me here.

Would something like this work?

```suggestion
Use `ThemeProvider` if you want to use a custom theme in your application.
If this provider is not used, Material UI components will use the library's default theme.
```